app/services/quotes_garden.js

```diff
@@ -2,17 +2,24 @@ const axios = require('axios');
 const logger = require('../logger');
 const config = require('../../config');
 const errors = require('../errors');
+const { DEFAULT_LENGTH_WEET } = require('../helpers/constants');
 
 const getRandomQuote = async () => {
   try {
     logger.info('quotes_garden-service::getRandomQuote');
-    const result = await axios.get(config.common.quoteGardenEndpointsApi.getRandomQuote);
-    const {
-      data: {
-        data: [payload]
-      }
-    } = result;
-    return payload;
+    let quoteText = null;
+    while (quoteText === null || quoteText.lenght > DEFAULT_LENGTH_WEET) {
```

Comment:
I don't get what are you doing here...
Do you check the randomQuote length and ask for a new one if it is bigger than needed?